packages/showcase/src/app/utility-classes/form/metadata.ts

```diff
@@ -0,0 +1,248 @@
+import { ComponentMetadata } from "../../../common/types";
+import { CSS } from "../../../common/html";
+
+const CONTROL_SLOT = "(Your control here)";
+
+export const formMetadata = {
+  title: "Form",
+  description:
+    "Form CSS classes provide consistent spacing and alignment, reduce repetitive code, and simplify maintenance—making forms easier to style and more polished across your app.",
+
+  codeSnippets: {
+    fieldBlock: {
+      linkId: "field-block",
+      title: "1. Field: Block",
+      template: {
+        tag: "div",
+        class: `${CSS.UTILS.FORM.FIELD.SELECTOR} ${CSS.UTILS.FORM.FIELD_BLOCK.SELECTOR}`,
+        children: [
+          {
+            tag: "label",
+            class: CSS.LABEL.SELECTOR,
+            properties: [{ name: "htmlFor", value: "control-id" }],
```

Comment:
Don't use `htmlFor`. That property only exists in JSX.

```suggestion
            properties: [{ name: "for", value: "control-id" }],
```